bindings/cron/cron.go

```diff
@@ -27,14 +27,14 @@ import (
 
 // Binding represents Cron input binding.
 type Binding struct {
-	logger        logger.Logger
-	name          string
-	schedule      string
-	parser        cron.Parser
-	runningCtx    context.Context
-	runningCancel context.CancelFunc
+	logger   logger.Logger
+	name     string
+	schedule string
+	parser   cron.Parser
 }
 
+var stopCh = make(map[string]chan bool)
```

Comment:
Please keep this as a property of the object and not a global variable

Reply:
https://github.com/dapr/components-contrib/pull/2191#discussion_r1016806187
It is because we cannot rely on context as property of the cron object itself